src/core/sync/event.d

```diff
@@ -0,0 +1,318 @@
+/**
+ * The event module provides a primitive for lightweight signalling of other threads
+ * (aka Windows Events)
```

Comment:
Reading this, I though the implementation was for Windows only.

Reply:
reworded slightly.